### PR TITLE
Fix CMakeLists

### PR DESCRIPTION
**Pull request by AlexanderVeselov**

### CODE REVIEW

File: Baikal/CMakeLists.txt [hunk 1]
```diff
@@ -3,36 +3,41 @@ set(CONTROLLERS_SOURCES
     Controllers/clw_scene_controller.h
     Controllers/scene_controller.h
     Controllers/scene_controller.inl
-    Controllers/scene_controller.cpp)
-    
-set(ESTIMATORS_SOURCES 
+    Controllers/scene_controller.cpp
+)
```
Comment (Kvalme) [inline Baikal/CMakeLists.txt]: Please, revert

**Review by Kvalme — CHANGES_REQUESTED** [review]
Please, revert all cosmetic changes

File: Baikal/CMakeLists.txt [hunk 2]
```diff
@@ -4,24 +4,24 @@ set(CONTROLLERS_SOURCES
     Controllers/scene_controller.h
     Controllers/scene_controller.inl
     Controllers/scene_controller.cpp)
-
```
Comment (Kvalme) [inline Baikal/CMakeLists.txt]: not reverted

**Review by Kvalme — CHANGES_REQUESTED** [review]

File: Baikal/CMakeLists.txt [hunk 3]
```diff
@@ -180,15 +184,44 @@ if (BAIKAL_EMBED_KERNELS)
...
-    
+
+# Add symbolic link from binary to Baikal/Kernels directory
+set(BAIKAL_KERNELS_DIR Baikal/Kernels)
+file(TO_NATIVE_PATH ${Baikal_SOURCE_DIR}/${BAIKAL_KERNELS_DIR} BAIKAL_KERNELS_SRC)
+file(TO_NATIVE_PATH ${CMAKE_BINARY_DIR}/${BAIKAL_KERNELS_DIR} BAIKAL_KERNELS_DST)
+
+if (WIN32)
+    add_custom_target(BaikalKernelsDir ALL
+        COMMAND IF NOT EXIST ${BAIKAL_KERNELS_DST} mklink /D ${BAIKAL_KERNELS_DST} ${BAIKAL_KERNELS_SRC}
```
Comment (Kvalme) [inline Baikal/CMakeLists.txt:209]: cmake -E create_symlink don't work on windows?

File: BaikalTest/CMakeLists.txt [hunk 4]
```diff
@@ -12,43 +12,38 @@ set(SOURCES
...
 set_target_properties(BaikalTest
-    PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY ${Baikal_SOURCE_DIR}/BaikalTest)
+    PROPERTIES VS_DEBUGGER_WORKING_DIRECTORY ${Baikal_SOURCE_DIR}/BaikalTest FOLDER Baikal)
 target_compile_definitions(BaikalTest PRIVATE _SILENCE_TR1_NAMESPACE_DEPRECATION_WARNING=1)
 
-add_custom_target(BaikalTestImagesDir)
-
+# Add symbolic links to BaikalTest/OutputImages and BaikalTest/ReferenceImages directory
 set(BAIKALTEST_REFERENCEIMAGES_DIR ReferenceImages)
-FILE(TO_NATIVE_PATH ${Baikal_SOURCE_DIR}/BaikalTest/${BAIKALTEST_REFERENCEIMAGES_DIR} BAIKALTEST_REFERENCEIMAGES_SRC)
```
Comment (Kvalme) [inline BaikalTest/CMakeLists.txt]: please, revert

**Review by Kvalme — CHANGES_REQUESTED** [review]

File: BaikalTest/CMakeLists.txt (hunk 4 above)
Comment (AlexanderVeselov) [reply BaikalTest/CMakeLists.txt]: Ok

File: Baikal/CMakeLists.txt (hunk 3 above)
Comment (AlexanderVeselov) [reply Baikal/CMakeLists.txt:209]: No, only Unix

**Comment by Kvalme** [comment]
Ok. Create external review

**Pull request closed without merging**